csharp/Adapter/Microsoft.Spark.CSharp/Services/LoggerServiceFactory.cs

```diff
@@ -31,7 +30,21 @@ public static void SetLoggerService(ILoggerService loggerServiceOverride)
         /// <returns>An instance of logger service</returns>
         public static ILoggerService GetLogger(Type type)
         {
-            return  loggerService.GetLoggerInstance(type);
+            return loggerService.Value.GetLoggerInstance(type);
+        }
+
+        /// <summary>
+        /// if there exists exe.config file, then use log4net
+        /// </summary>
+        /// <returns></returns>
+        private static ILoggerService GetDefaultLogger()
+        {
+            if (File.Exists(AppDomain.CurrentDomain.SetupInformation.ConfigurationFile))
```

Comment:
I've considered this case (has xxx.exe.config but no log4net setting nodes) , log4net will show a warning just like previous : (detail to see reply to your next question)
`log4net:ERROR Failed to find configuration section 'log4net' *** `
Then continue with default logger service.